fearless/Common/Model/KeystoreTag.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+
+enum KeystoreTag: String, CaseIterable {
+    case pincode
+
+    static func secretKeyTagForAddress(_ address: String) -> String { address + "-" + "secretKey" }
+    static func entropyTagForAddress(_ address: String) -> String { address + "-" + "entropy" }
+    static func deriviationTagForAddress(_ address: String) -> String { address + "-" + "deriv" }
+    static func seedTagForAddress(_ address: String) -> String { address + "-" + "seed" }
```

Comment:
It's a matter of style and consistency, but if we define suffixes as "-substrateSecretKey" later, we probably don't need extra concatenation here

Reply:
Actually, I am planing to switch `KeystoreTag -> KeystoreTagV1` and `KeystoreTag -> KeystoreTagV2` when all preparations are done